python/semantic_kernel/agents/autogen/autogen_conversable_agent.py

```diff
@@ -36,6 +38,78 @@
 logger: logging.Logger = logging.getLogger(__name__)
 
 
+@experimental
+class AutoGenConversableAgentThread(AgentThread):
+    """Azure AI Agent Thread class."""
+
+    def __init__(self, chat_history: ChatHistory | None = None, thread_id: str | None = None) -> None:
+        """Initialize the AutoGenConversableAgentThread Thread.
+
+        Args:
+            chat_history: The chat history for the thread. If None, a new ChatHistory instance will be created.
+            thread_id: The ID of the thread. If None, a new thread will be created.
+        """
+        self._chat_history = chat_history or ChatHistory()
+        self._is_active = thread_id is not None
+        self._thread_id = thread_id
+
+    @override
+    @property
+    def is_active(self) -> bool:
+        """Indicates whether the thread is currently active."""
+        return self._is_active
```

Comment:
why not skip the `_is_active` and just do:
```suggestion
        return self._thread_id is not None
```